Q/Qt/build_tarballs.jl

```diff
@@ -0,0 +1,154 @@
+        rsync -a usr/* /opt/$target/$target/sys-root/usr/
+        cp -a System /opt/$target/$target/sys-root/
+        
+        cd $WORKSPACE/srcdir/qt-everywhere-src-*/qtbase
+        
+        cat <<EOT > mkspecs/features/mac/default_pre.prf
+CONFIG = asset_catalogs rez \$\$CONFIG
+load(default_pre)
+
+QMAKE_ASSET_CATALOGS_APP_ICON = AppIcon
+
+macx-xcode:qtConfig(static): \\
+    QMAKE_XCODE_DEBUG_INFORMATION_FORMAT = dwarf
+
+QMAKE_XCODE_LIBRARY_SUFFIX_SETTING = QT_LIBRARY_SUFFIX
+
+xcode_copy_phase_strip_setting.name = COPY_PHASE_STRIP
+xcode_copy_phase_strip_setting.value = NO
+QMAKE_MAC_XCODE_SETTINGS += xcode_copy_phase_strip_setting
+EOT
+        
+        sed -i '1s;^;QMAKE_MAC_SDK.macosx.Path = '"/opt/$target/$target/sys-root"'\
+        QMAKE_MAC_SDK.macosx.SDKVersion = '"10.14"'\
+        QMAKE_MAC_SDK.macosx.PlatformPath = '"/opt/$target"'\n;' 'mkspecs/features/mac/sdk.prf'
+        echo "" >  mkspecs/features/mac/no_warn_empty_obj_files.prf
+        
+        sed -i 's!-fuse-ld=x86_64-apple-darwin14!-fuse-ld=/opt/bin/x86_64-apple-darwin14-ld!g' /opt/bin/x86_64-apple-darwin14-clang++
+        sed -i 's!-fuse-ld=x86_64-apple-darwin14!-fuse-ld=/opt/bin/x86_64-apple-darwin14-ld!g' /opt/bin/x86_64-apple-darwin14-clang
+        
+        cd $WORKSPACE/srcdir/build
+        apk add g++ linux-headers
+        export PATH=$(echo "$PATH" | sed -e 's!/opt/bin:!!')
+        export QT_MAC_SDK_NO_VERSION_CHECK=1
+        ../qt-everywhere-src-*/configure \
+            QMAKE_CXXFLAGS+=-F/opt/$target/$target/sys-root/System/Library/Frameworks \
+            QMAKE_RANLIB=/opt/bin/ranlib \
+            QMAKE_MACOSX_DEPLOYMENT_TARGET=10.14 \
+            -platform linux-g++ -xplatform macx-clang -device-option CROSS_COMPILE=/opt/bin/$target- \
+            -prefix $prefix $commonoptions \
+            -skip qtwinextras
+        ;;
+        
+	*w64-mingw*)
+        apk add g++ linux-headers
+        export PATH=$(echo "$PATH" | sed -e 's!/opt/bin:!!')
+        
+        ../qt-everywhere-src-*/configure -platform linux-g++ -xplatform win32-g++ -device-option CROSS_COMPILE=/opt/bin/$target- \
+            -prefix $prefix $commonoptions \
+            -opengl desktop
+		;;
+
+    *arm-linux*)
+        apk add g++ linux-headers
+        export PATH=$(echo "$PATH" | sed -e 's!/opt/bin:!!')
+        export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$prefix/share/pkgconfig
+        export PKG_CONFIG_LIBDIR=$prefix/lib/pkgconfig
+        
+        ../qt-everywhere-src-*/configure QMAKE_LFLAGS=-liconv -platform linux-g++ -device linux-rasp-pi3-g++ -device-option CROSS_COMPILE=/opt/bin/$target- \
+            -extprefix $prefix $commonoptions \
+            -skip qtwinextras -system-xcb -fontconfig -sysroot /opt/$target/bin/../$target/sys-root
+		;;
+esac
+
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
```

Comment:
Any chance of getting Qt on all supported platforms in the future?

Reply:
Let's try that once the current list works ;)

Unfortunately so far it turns out that each platform requires its own tweaks, so I'm not expecting a picnic.